lib/SimpleHttpClient/Communicator.cpp

```diff
@@ -141,6 +141,8 @@ Communicator::Communicator() : _curl(nullptr), _mc(CURLM_OK), _enabled(true) {
     THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_OUT_OF_MEMORY, "unable to initialize curl");
   }
   
+  curl_multi_setopt(_curl, CURLMOPT_MAXCONNECTS, 20);
```

Comment:
Why is this sensible? The default of adding 4 for each added easy handle seems better to me. In case of large clusters this will ruin performance, no?

Reply:
I added this because I saw curl closing connections because the connection cache was full, and all that while running simple `shell_client` tests. So it seems the default leads to connections being closed and re-opened a lot of times.